src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/Metadata/IModelBindingMessageProvider.cs

```diff
@@ -30,5 +30,46 @@ public interface IModelBindingMessageProvider
         /// </summary>
         /// <value>Default <see cref="string"/> is "The value '{0}' is invalid.".</value>
         Func<string, string> ValueMustNotBeNullAccessor { get; }
+
+        /// <summary>
+        /// Fallback error message HTML and tag helpers display when a property is invalid but the
+        /// <see cref="ModelError"/>s have <c>null</c> <see cref="ModelError.ErrorMessage"/>s.
+        /// </summary>
+        /// <value>Default <see cref="string"/> is "The value '{0}' is invalid.".</value>
+        Func<string, string> ValueInvalid_UnknownErrorResource { get; }
+
+        /// <summary>
+        /// Replacement <see cref="ModelError.ErrorMessage"/> used in <see cref="ModelError"/> when
```

Comment:
`Error message the model binding system adds when `.... rather than what you have